runtime/colors/tools/check_colors.vim

```diff
@@ -1,238 +1,209 @@
-" This script tests a color scheme for some errors and lists potential errors.
-" Load the scheme and source this script, like this:
-"    :edit colors/desert.vim | :so colors/tools/check_colors.vim
+vim9script
+# This script tests a color scheme for some errors and lists potential errors.
+# Load the scheme and source this script, like this:
+#    :edit colors/desert.vim | :ru colors/tools/check_colors.vim
 
-let s:save_cpo= &cpo
+var save_cpo = &cpo
 set cpo&vim
```

Comment:
You don't need to save, set and restore `cpoptions` in a `vim9script` file. See the third paragraph under `:h E1101`.